Code/GraphMol/SmilesParse/SmilesWrite.h

```diff
@@ -39,6 +39,8 @@ struct RDKIT_SMILESPARSE_EXPORT SmilesWriteParams {
   bool includeDativeBonds =
       true; /**< include the RDKit extension for dative bonds. Otherwise dative
                bonds will be written as single bonds*/
+  bool ignoreAtomMapNumbers = false; /**< If true, ignores any atom map numbers
+                                        when generating canonical SMILES */
```

Comment:
The original formulation almost makes it sound like the atom map numbers will not show up in the SMILES
```suggestion
  bool ignoreAtomMapNumbers = false; /**< If true, ignores any atom map numbers
                                        when canonicalizing the molecule */
```